instat/ucrAxes.vb

```diff
@@ -96,21 +97,22 @@ Public Class ucrAxes
         bInitialiseControls = True
     End Sub
 
-    Public Sub SetRCodeForControl(bIsXAxis As Boolean, Optional clsNewXYScaleContinuousFunction As RFunction = Nothing, Optional clsNewXYlabTitleFunction As RFunction = Nothing, Optional clsNewBaseOperator As ROperator = Nothing, Optional bReset As Boolean = False)
+    Public Sub SetRCodeForControl(bIsXAxis As Boolean, Optional strNewAxisType As String = "Continuous", Optional clsNewXYScaleContinuousFunction As RFunction = Nothing, Optional clsNewXYlabTitleFunction As RFunction = Nothing, Optional clsNewBaseOperator As ROperator = Nothing, Optional bReset As Boolean = False)
         bRCodeSet = False
         If Not bInitialiseControls Then
             InitialiseControl()
         End If
         clsBaseOperator = clsNewBaseOperator
         bIsX = bIsXAxis
-
+        strAxisType = strNewAxisType
         If bIsXAxis Then
             bIsX = True
             strAxis = "x"
         Else
             bIsX = False
             strAxis = "y"
         End If
+        ucrInputAxisType.SetName(strAxisType)

```

Comment:
New control which shows the scales type. This controls which controls are visible. The value is passed through from the sub dialog but can also be changed by the user if needed (but not recommended)